### PR TITLE
nl language (dutch)

### PR DESCRIPTION
**Pull request by thijsi123**
I hope that this is what you meant, these are the Dutch translation files in short nl.

### CODE REVIEW

**Comment by thijsi123** [comment]
By the way, sorry for the messy commits, I had a few problem renaming the folders and I already had changed the normal language to dutch, I reversed th normal language back to English, and added the values-nl folder.

File: app/src/main/res/values-nl/translation.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    <!-- Language Names Here -->
+    <string-array name="languages">
+        <item>English</item>
+        <item>Dutch </item>
+    </string-array>
+
+    <!-- Edit your name here according to translated language -->
+    <string name="English_contributors">Vaibhav</string>
+    <string name="Dutch_contributors">Thijsi123</string>
```
Comment (dead8309) [inline app/src/main/res/values-nl/translation.xml:12]: String `name` fields should start with lower case characters as per naming convention

File: app/src/main/res/values-nl/translation.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+
+
+    <!-- Don't Edit this -->
+    <string-array name="contributors">
+        <item>@string/english_contributors</item>
```
Comment (dead8309) [inline app/src/main/res/values-nl/translation.xml:18]: You have to add the string name field to this string array
Like 
```xml
<item>@string/dutch_contributors</item>

**Comment by dead8309** [comment]
> By the way, sorry for the messy commits, I had a few problem renaming the folders and I already had changed the normal language to dutch, I reversed th normal language back to English, and added the values-nl folder.

`colors.xml` , `themes.xml` and `translation.xml` are not required to be added to other directories.

Why did you deleted `strings.xml` ?

Close this pr and make a new one with all the changes requested or send the translations file in discord channel and I'll make the necessary changes and add it.

**Comment by thijsi123** [comment]
I'll do it as soon as I can, thank you.

**Pull request closed without merging**